assets/js/apps/search/components/FilterSidebar.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
import React from "react"
```